### PR TITLE
Fixed parameters for build.complete and ms.vss-pipelines.stage-state-changed-event, .yaml file extension bug, some code cleanup

### PR DESCRIPTION
**Pull request by KevinIsMyName**
- Updated README.md with a FAQ
 - Fixed single-quotes linting in bundle.js
 - Moved list of available webhook-trigger-event-types into README.md instead of as a code comment in EventTypes.ts
 - Updated Azure API version from 1.0 to 5.1-preview for consistency (web UI uses 5.1-preview).
 - Fixed the init functions to create .yaml files instead of .undefined files
 - Removed redundant spinner references for code-cleanup.
 - Added possible values for some webhook-trigger-event-type arguments

### CODE REVIEW

**Comment by KevinIsMyName** [comment]
Any reason why it shows ALL commits? Shouldn't it only show the commits since last change, which is when my PR was merged?

File: README.md [hunk 1]
```diff
@@ -10,3 +10,19 @@ az extension add --name azure-devops
 az devops configure --defaults organization="https://dev.azure.com/<ORGANIZATION>" project="<PROJECT>"
 node .
 ```
+
+
+## FAQ
```
Comment (wgoehrig) [inline README.md:15]: Can we explain this is specific to the svc-hooks stuff?

File: src/svc-hooks/create-init.ts [hunk 2]
```diff
@@ -22,8 +21,9 @@ export function handler(argv: any) {
       eventType: "build.complete",
       url: null,
       eventSpecificArgs: {
-        definitionName: null,
-        buildStatus: null,
+        pipelineName: "'' | <pipeline-name>",
```
Comment (wgoehrig) [inline src/svc-hooks/create-init.ts:24]: Can we leave these as null but add the options as comments?  That way the YAML still parses and tokenized correctly by editors...

File: src/svc-hooks/create.ts [hunk 3]
```diff
@@ -41,35 +41,40 @@ export async function handler(argv: any) {
...
-            definitionName: hook.eventSpecificArgs.definitionName,
             buildStatus: hook.eventSpecificArgs.buildStatus,
+            definitionName: hook.eventSpecificArgs.pipelineName,
+            projectId: await searchProjId(hook.org, hook.project),
           };
           break;
         case "ms.vss-pipelines.stage-state-changed-event":
           hookDataFormatted.publisherInputs = {
-            pipelineId: hook.eventSpecificArgs.pipelineId,
+            pipelineId: hook.eventSpecificArgs.buildDefinitionId, // TODO: Figure out which input is best and how to get pipelineId from it.
```
Comment (wgoehrig) [inline src/svc-hooks/create.ts:65]: I'm confused what this TODO is supposed to mean...

**Review by wgoehrig — CHANGES_REQUESTED** [review]

File: src/svc-hooks/create.ts (hunk 3 above)
Comment (KevinIsMyName) [reply src/svc-hooks/create.ts:65]: Whoops, this is a leftover comment for me when I was figuring out the buildDefinitionId input, I'll remove this. Not necessary.

**Comment by wgoehrig** [comment]
> Any reason why it shows ALL commits? Shouldn't it only show the commits since last change, which is when my PR was merged?
> 
> I only made two commits, they are both after commit: _Merge branch 'main' of https://github.com/KevinIsMyName/az-devops-tools [1c02934](https://github.com/wgoehrig/az-devops-tools/commit/1c029346732be74c2370a4ab1e5db1136bd8b885)_
> 
> Or can this be fixed by deleting my fork, and re-forking?

It's showing all commits because I squashed your fork into main when I merged the last PR.  You _could_ fix this by creating a new fork or just rebasing your current fork, but since I'm going to squash this PR too there's no need.

**Review by wgoehrig — APPROVED** [review]

**Review by KevinIsMyName — COMMENTED** [review]
some text here, not really sure how this works

oh wait this is from the reviewer's pov

**Comment by KevinIsMyName** [comment]
I force pushed because of a `git commit --amend --no-edit` after I just pushed. Nothing wrong to see here.

**Pull request merged**